kiwi/builder/container.py

```diff
@@ -80,13 +103,30 @@ def create(self):
 
         * image="docker"
         """
-        log.info(
-            'Setting up %s container', self.requested_container_type
-        )
-        container_setup = ContainerSetup(
-            self.requested_container_type, self.root_dir, self.container_config
-        )
-        container_setup.setup()
+        if not self.base_image:
+            log.info(
+                'Setting up %s container', self.requested_container_type
+            )
+            container_setup = ContainerSetup(
+                self.requested_container_type, self.root_dir,
+                self.container_config
+            )
+            container_setup.setup()
+        else:
+            try:
+                checksum = Checksum(self.base_image)
+                with open(self.base_image_md5) as md5_file:
+                    base_image_md5 = md5_file.read()
+            except Exception as e:
+                raise KiwiContainerBuilderError(
+                    '%s: %s' % (type(e).__name__, format(e))
```

Comment:
We should stop using old formatters. They are gone with 2.6 version and we have `.format(...)` instead. Let's do this:
```python
raise KiwiContainerBuilderError('{name}: {reason}'.format(name=type(e).__name__,
                                                          reason=format(e)))